challenge3/webapp/caesaris/app/lib/data_process.rb

```diff
@@ -0,0 +1,77 @@
+class DataProcess
+  class << self
+    # calc average energy_production per house for cities
+    # res: { "Oxford" => { "2011-7" => [min, ave, max] } }
+    def house_energy_prod_time_series
+      results = {}
+      City.all.each do |city|
```

Comment:
```suggestion
      City.all.each_with_object({}) do |city, results|
```

`each_with_object` を使うと、6行目や、25行目を書く必要がない。

Reply:
完全リファクタリングしました
https://github.com/jerrywdlee/EnergyDataSimulationChallenge/commit/84adc94aa037a4c673b97b84b8ed7c14c2a67c7e